test/functional/cmdLineTests/criu/src/org/openj9/criu/CRIUTestUtils.java

```diff
@@ -78,4 +78,24 @@ public static void checkPointJVM(Path path, boolean deleteDir) {
 			System.err.println("CRIU is not enabled");
 		}
 	}
+
+	public static void checkPointJVM(CRIUSupport criu, Path path, boolean deleteDir) {
```

Comment:
can you merge this with the one above? if Criu is null we can create a new one, else we can use the existing one

Reply:
Modified the method above to invoke this one instead, `new CRIUSupport(path)` requires the existence of `path`, i.e., after `createCheckpointDirectory(path)`.